File: c#/removeVowels.cs
    public static string RemoveVowels(string str) 
    /* Create a function that takes a string and returns a new string with all vowels removed.*/
    {
        StringBuilder result = new StringBuilder();
        char[] vowels = ['a','e','i','o','u'];

        foreach (char c in str) {

            if (Array.IndexOf(vowels, c) == -1) {

                result.Append(c);

                
            }


        

      
    }

    return result.ToString();
        
    }

File: python/removeVowels.py
"""
Create a function that takes a string and returns a new string with all vowels removed.

"""


def remove_vowels(txt):
    vowels = 'aeiouAEIOU'
    result = ''
    length = len(txt)
    for i in range(0,length):
        if txt[i] not in vowels:
            result += txt[i]

    return result